test/test_rosbag/test/test_bag.py

```diff
@@ -312,6 +312,14 @@ def test_get_time(self):
             self.assertEquals(start_stamp, 0.0)
             self.assertEquals(end_stamp, 99.0)
 
+    def test_get_time_empty_bag(self):
+        """Test for issue #657"""
+        fn = '/tmp/test_get_time_emtpy_bag.bag'
```

Comment:
That would be better, but all of the other tests in this file do the same as @galou is doing. So it's probably ok.